samples/account-linking/csharp/.tours/messaging-extension-account-linking.tour

```diff
@@ -0,0 +1,107 @@
+{
+  "$schema": "https://aka.ms/codetour-schema",
+  "title": "Messaging Extension Account Linking",
+  "steps": [
+    {
+      "file": "Source/Bots/SampleActivityHandler.cs",
+      "description": "Account linking with messaging extensions is a bit easier than tabs. For this sample, we demonstrate how to do this with a [messaging extension search command](https://docs.microsoft.com/en-us/microsoftteams/platform/messaging-extensions/how-to/search-commands/define-search-command). \r\n\r\nThe flow will be nearly identical with all messaging extensions so this should be adaptable for [action commands](https://docs.microsoft.com/en-us/microsoftteams/platform/messaging-extensions/how-to/action-commands/define-action-command?tabs=AS) or [link unfurling](https://docs.microsoft.com/en-us/microsoftteams/platform/messaging-extensions/how-to/link-unfurling?tabs=dotnet).",
+      "line": 65
+    },
+    {
+      "file": "Source/Bots/SampleActivityHandler.cs",
+      "description": "As before we get the user's id (Azure Ad id, not Bot Framework id) and their tenant id. Unlike in the tab flow, these two values are trustworthy in the inbound payload so we don't need an Azure AD access token to authenticate the user.\r\n\r\n",
+      "line": 71,
+      "selection": {
+        "start": {
+          "line": 70,
+          "character": 9
+        },
+        "end": {
+          "line": 71,
+          "character": 67
+        }
+      }
+    },
+    {
+      "file": "Source/Bots/SampleActivityHandler.cs",
+      "description": "First, lets trace what will happen when the user hasn't linked their GitHub account. The `OAuthTokenProvider` described in the [Token Storage][#Token Storage] tour will return a `NeedsConsentResult` if it cannot find the user's token in its database. ",
```

Comment:
> [Token Storage][#Token Storage]
Is this supposed to be a link? It isn't showing for me at least

Reply:
It is supposed to be a link to the Token Storage tour. All the tours are some sort of messed up right now. Unsure why it seems to work on my machine but nowhere else.